src/context/AppProvider/AppContext.tsx

```diff
@@ -61,6 +65,31 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
   const routeAssetId = useRouteAssetId()
   const isSnapInstalled = Boolean(useIsSnapInstalled())
   const previousIsSnapInstalled = usePrevious(isSnapInstalled)
+  const { close: closeModal, open: openModal } = useModal('ledgerOpenApp')
+
+  useEffect(() => {
+    const handleLedgerOpenApp = ({ chainId, isSigning, reject }: LedgerOpenAppEventArgs) => {
+      const onCancel = () => {
+        closeModal()
+        reject()
+      }
+
+      openModal({ chainId, onCancel, isSigning })
```

Comment:
I wasn't able to see any occurence of Ledger open app gate modal with `isSigning` div:

<img width="579" alt="image" src="https://github.com/user-attachments/assets/ca5deebb-43a9-4e2f-b150-c39617e5f0e3">
<img width="659" alt="image" src="https://github.com/user-attachments/assets/abcff89e-53aa-4fba-bf5b-a8c08260c087">
<img width="756" alt="image" src="https://github.com/user-attachments/assets/1dceca70-8f30-4427-8a45-30b3e165dece">
<img width="556" alt="image" src="https://github.com/user-attachments/assets/3ab23616-aa35-4284-ac45-24ca029a5af8">

Reply:
Yeah, this is due to the fact we call `getAddress` before signing in (almost) every occurrence. Updating the warning to be generic and always shown.

Reply:
Generic device prompt messaging (we can update as needed)
![image](https://github.com/user-attachments/assets/dcdd4cca-34fc-48ae-9bf1-71acc8c87b35)